src/services/membership_api/routes/memberRoute.js

```diff
@@ -0,0 +1,8 @@
+const express = require('express');
+const router = express.Router();
+
+router.get('/hello_members', (req, res) => {
+  res.send('Hello members');
+});
+
+module.exports = router;
```

Comment:
use **ES6 export by default** and also learn when to use **module.exports**, **export** and **export by default**

Reply:
changes made

Reply:
done